drivers/hmis/app/views/hmis_client/services/_table.haml

```diff
@@ -0,0 +1,27 @@
+.table-responsive
+  %table.table
+    %thead
+      %tr
+        %th
+        %th Project
+        %th Entry Date
+        %th Date of Service
+        %th Service Type
+        %th
+    %tbody
+      - custom_services.each do |service|
+        %tr
+          %td
+            .ds{class: "ds-color-#{service.data_source_id}"}= ds_short_name_for(service.client.id)
+          %td= service.project.name
+          %td= service.enrollment.entry_date
+          %td= service.date_provided
+          %td= service.display_name
+          %td.text-right
+            - if @client.hmis_source_visible_by?(current_user)
+              = link_to 'HMIS', service.data_source.hmis_url_for(service, user: current_user), class: 'btn btn-xs btn-secondary btn-muted btn-hmis mb-2'
```

Comment:
I think this should be the "Open in HMIS" link style,. since that is always what we use to link to HMIS. This button style usually links to a different warehouse page.